mmv1/products/compute/api.yaml

```diff
@@ -2592,6 +2689,101 @@ objects:
           Only ROUND_ROBIN and RING_HASH are supported when the backend service is referenced
           by a URL map that is bound to target gRPC proxy that has validate_for_proxyless
           field set to true.
+      - !ruby/object:Api::Type::Array
+        name: 'localityLbPolicies'
+        description: |
+          A list of locality load balancing policies to be used in order of
+          preference. Either the policy or the customPolicy field should be set.
+          Overrides any value set in the localityLbPolicy field.
+
+          localityLbPolicies is only supported when the BackendService is referenced
+          by a URL Map that is referenced by a target gRPC proxy that has the
+          validateForProxyless field set to true.
+        item_type: !ruby/object:Api::Type::NestedObject
+          name: 'localityLbPolicyConfig'
+          description: |
+            Container for either a built-in LB policy supported by gRPC or Envoy or
+            a custom one implemented by the end user.
+          properties:
+            - !ruby/object:Api::Type::NestedObject
+              name: 'policy'
+              exactly_one_of:
+                - policy
+                - customPolicy
+              description: |
+                The configuration for a built-in load balancing policy.
+              properties:
+                - !ruby/object:Api::Type::Enum
+                  name: 'name'
```

Comment:
Should this be required just like the field above?

Reply:
Yes! The region backend has the same requirement. Updated.